lib/logging/GhostLogger.js

```diff
@@ -234,11 +234,11 @@ class GhostLogger {
                     statusCode: err.statusCode,
                     level: err.level,
                     message: err.message,
-                    context: err.context,
+                    context: jsonStringifySafe(err.context),
```

Comment:
Read in the documentation for bunyan serializers, they [should be](https://www.npmjs.com/package/bunyan#requirements-for-serializers-functions) defensive and this seemed like a perfect solution for that

Reply:
🤔 I don't understand. Why is it better to use jsonStringifySafe and not JSON.stringify()?

Reply:
`jsonStringifySafe` protects from throwing an error when there's a circular dependency in the stringified object and if we need to be defensive in our code it's a way to go. Also, was used here for a similar reason - https://github.com/TryGhost/Ignition/blob/master/lib/logging/GhostLogger.js#L328-L329

Reply:
Yeah i know, but why would there be a circular dependency in context or error details 🤔
I am fine with using it for it, but the reason is not 100% clear to me.

> Also, was used here for a similar reason - 

Yeah but because of express's `req` object, which contains function references etc.

Reply:
My reasoning is if we don't have 100% control over what gets passed into a method we should always assume for the worst case scenario. It's unlikely such an object will be passed here, but if it has too be 100% bulletproof that's the code we need here :smiley:  Maybe someone will decide to pass in a `req` object as an errorDetail :thinking: or there will be a weird object structure with circular dependencies that will be used in future ghost code :woman_astronaut:

Reply:
> we should always assume for the worst case scenario

Ok agree :) Let's use it, it's probably more flexible.